tests/test_22_04_2025.py

```diff
@@ -0,0 +1,984 @@
+
+    # Voting Permissions Transition
+    UNSAFELY_MODIFY_VOTE_TIME_ROLE = web3.keccak(text="UNSAFELY_MODIFY_VOTE_TIME_ROLE")
+    assert not acl.hasPermission(VOTING, VOTING, UNSAFELY_MODIFY_VOTE_TIME_ROLE)
+    assert not acl.hasPermission(AGENT, VOTING, UNSAFELY_MODIFY_VOTE_TIME_ROLE)
+    assert acl.getPermissionManager(VOTING, UNSAFELY_MODIFY_VOTE_TIME_ROLE) == ZERO_ADDRESS
+
+    # TokenManager Permissions Transition
+    MINT_ROLE = web3.keccak(text="MINT_ROLE")
+    assert not acl.hasPermission(VOTING, TOKEN_MANAGER, MINT_ROLE)
+    assert not acl.hasPermission(AGENT, TOKEN_MANAGER, MINT_ROLE)
+    assert acl.getPermissionManager(TOKEN_MANAGER, MINT_ROLE) == ZERO_ADDRESS
+
+    REVOKE_VESTINGS_ROLE = web3.keccak(text="REVOKE_VESTINGS_ROLE")
+    assert not acl.hasPermission(VOTING, TOKEN_MANAGER, REVOKE_VESTINGS_ROLE)
+    assert not acl.hasPermission(AGENT, TOKEN_MANAGER, REVOKE_VESTINGS_ROLE)
+    assert acl.getPermissionManager(TOKEN_MANAGER, REVOKE_VESTINGS_ROLE) == ZERO_ADDRESS
+
+    BURN_ROLE = web3.keccak(text="BURN_ROLE")
+    assert not acl.hasPermission(VOTING, TOKEN_MANAGER, BURN_ROLE)
+    assert not acl.hasPermission(AGENT, TOKEN_MANAGER, BURN_ROLE)
+    assert acl.getPermissionManager(TOKEN_MANAGER, BURN_ROLE) == ZERO_ADDRESS
+
+    ISSUE_ROLE = web3.keccak(text="ISSUE_ROLE")
+    assert not acl.hasPermission(VOTING, TOKEN_MANAGER, ISSUE_ROLE)
+    assert not acl.hasPermission(AGENT, TOKEN_MANAGER, ISSUE_ROLE)
+    assert acl.getPermissionManager(TOKEN_MANAGER, ISSUE_ROLE) == ZERO_ADDRESS
+
+    # Finance Permissions Transition
+    CHANGE_PERIOD_ROLE = web3.keccak(text="CHANGE_PERIOD_ROLE")
+    assert not acl.hasPermission(VOTING, FINANCE, CHANGE_PERIOD_ROLE)
+    assert not acl.hasPermission(AGENT, FINANCE, CHANGE_PERIOD_ROLE)
+    assert acl.getPermissionManager(FINANCE, CHANGE_PERIOD_ROLE) == ZERO_ADDRESS
+
+    CHANGE_BUDGETS_ROLE = web3.keccak(text="CHANGE_BUDGETS_ROLE")
+    assert not acl.hasPermission(VOTING, FINANCE, CHANGE_BUDGETS_ROLE)
+    assert not acl.hasPermission(AGENT, FINANCE, CHANGE_BUDGETS_ROLE)
+    assert acl.getPermissionManager(FINANCE, CHANGE_BUDGETS_ROLE) == ZERO_ADDRESS
+
+    # EVMScriptRegistry Permissions Transition
+    REGISTRY_MANAGER_ROLE = web3.keccak(text="REGISTRY_MANAGER_ROLE")
+    assert acl.hasPermission(VOTING, EVM_SCRIPT_REGISTRY, REGISTRY_MANAGER_ROLE)
+    assert not acl.hasPermission(AGENT, EVM_SCRIPT_REGISTRY, REGISTRY_MANAGER_ROLE)
+    assert acl.getPermissionManager(EVM_SCRIPT_REGISTRY, REGISTRY_MANAGER_ROLE) == VOTING
+
+    REGISTRY_ADD_EXECUTOR_ROLE = web3.keccak(text="REGISTRY_ADD_EXECUTOR_ROLE")
+    assert acl.hasPermission(VOTING, EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE)
+    assert not acl.hasPermission(AGENT, EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE)
+    assert acl.getPermissionManager(EVM_SCRIPT_REGISTRY, REGISTRY_ADD_EXECUTOR_ROLE) == VOTING
+
+    # CuratedModule Permissions Transition
```

Comment:
Add checks for all addresses in "Granted" column from [permissions transition table](https://github.com/lidofinance/dual-governance/blob/a22a141ba331fa2407f13711126cb9d3691c4f83/docs/permissions-transition/permissions-transition-hoodi.md#%EF%B8%8Fcuratedmodule0x5cdbe1590c083b5a2a64427faa63a7cfdb91fbb5)

Reply:
Same for the "Simple DVT Module Permissions Transition"